osc/osc.go

```diff
@@ -568,6 +572,18 @@ func (s *Server) Serve(c net.PacketConn) error {
 	}
 }
 
+// CloseConnection forcibly closes a server's connection.
+//
+// This causes a "use of closed network connection" error the next time the
```

Comment:
I have a feeling I found a way around this before but I can't remember it now 🤔 
I think it involved using the read timeout value, but I wouldn't worry about working it out for the save of the tests.